tools/extract-messages.js

```diff
@@ -54,12 +54,13 @@ Result.prototype.toString = function() {
   return result;
 };
 
-function Process() {
+function Process(includeLineNumberInPositions) {
   // data is a simple key-value dict, where key
   this.data = {};
+  this.includeLineNumberInPositions = false;
```

Comment:
Why don't you respect the function arg value?

Reply:
Sorry, missed your comment :) 

Fat finger, i initially added the argument on the function for offering a CLI interface to enable it.

Then I changed my mind, low chance other user than me need this, and it's fast to change hard-coded `false` to `true` in the code. And I forgot to remove the argument in the prototype .... I will remove it in the next translations PR